Adding loki, logging, and grafana as apps.

computate opened the pull request.

computate commented:
Does this require any more configuration to automatically apply these new apps on nerc-ocp-infra?

dystewart commented:
@computate If auto-sync is enabled in argocd then you wont have to do anything to apply but if not you'll need to manually sync once merged

computate commented on .gitignore:
```diff
@@ -0,0 +1 @@
+.project
```
@dystewart done.

larsks replied:
Note that you can create a local `.gitignore` file by putting things to ignore in `.git/info/exclude`.

larsks replied:
...and as long as I'm commenting here, rather than creating a new "Removed .gitignore" commit in this pull request, consider just updating the original commit and updating the PR.

larsks replied:
...and as long as I'm commenting here, rather than creating a new "Removed .gitignore" commit in this pull request, consider just updating the original commit and updating the PR.

naved001 commented on clusters/nerc-ocp-infra/logging/application.yaml:
```diff
@@ -0,0 +1,19 @@
+    path: logging/overlays/nerc-ocp-infra
+  destination:
+    name: nerc-ocp-infra
+    namespace: openshift-logging
```
This namespace is in PR https://github.com/OCP-on-NERC/nerc-ocp-config/pull/70 

But that namespace will be created by this argocd App, but we can't create this argocd app because it will look for the `openshift-logging` namespace to begin with. 

I am not sure sure about how much the namespace matters; we could put all these apps into the `openshift-gitops` namespace just so we have all the argocd apps in one namespace. 

Or we should be creating the namespace (and other clusterwide things) using the cluster-scope app which already exists. That's what we should be doing regardless of this situation. 

@larsks thoughts?

larsks replied:
These should live in their own namespace, which should be created via a PR to the `nerc-ocp-config` repository before this one merges.

larsks replied:
These should live in their own namespace, which should be created via a PR to the `nerc-ocp-config` repository before this one merges.

larsks reviewed (commented):
This PR doesn't modify `clusters/nerc-ocp-infra/kustomization`, which means none of these applications will actually be deployed by argocd. Is that intentional (that is, is this just to get the manifests in place and a future pr will activate them) or is that an oversite?

computate commented:
@larsks I forgot to add the new apps to the list in `clusters/nerc-ocp-infra/kustomization`, so I have added them now, I also added a separate pr in nerc-ocp-config to create the namespaces before the app: 
https://github.com/OCP-on-NERC/nerc-ocp-config/pull/72
